app/routes.py

```diff
@@ -11,11 +14,15 @@ def index():
         link=form.link.data
         d=Down(link)
         d.clear_folder()
-        d.dl()
+        # d.dl()
         # return send_from_directory('download', d.title+'.mp4', as_attachment=True)
         # return send_file(d.title+".mp4", as_attachment=True)
-        return send_file(f"download/{d.title}.mp4", as_attachment=True)
-        # return redirect(url_for("download", title=d.title+'.mp4'))
+        # return send_file(f"download/{d.title}.mp4", as_attachment=True)
+        print(f'Link: {d.dl_link()}')
+
+        response = Response(pytube.request.stream(d.dl_link()), mimetype='video/mp4')
```

Comment:
I should've probably moved this `pytube.request.stream()` call into a method in the Down class. Something like `dl_stream`, that way pytube is still only imported there.